sprocs/variants_select.sql

```diff
@@ -34,6 +41,11 @@ BEGIN
     WHERE
         q.id = variant_with_id.question_id;
 
+    IF variant_with_id.instance_question_id IS NOT NULL THEN
```

Comment:
How can `variant_with_id.instance_question_id` ever be NOT NULL? I can't see where it would get set above.

I also can't see where `variant_with_id.course_instance_id` (line 27) would be set?

Reply:
instance_question_id and course_instance_id come from v.* in the query. They are null when the issue is caused by an instructor in question preview (e.g., a syntax error in server.py).

Reply:
D'oh! Thank you for explaining to those of us who are a bit slow 😄